react-js/styling-react.md

```diff
@@ -0,0 +1,250 @@
+# Styling in React
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Guided Practice: 20 mins.
+
+### Prerequisites
+- [CSS Lesson](/css/css)
```

Comment:
These three are broken. 1. The files should probably be named .md. 2. search for the file name, such as css.md, to see if the path was changed.  I think css and bootstrap are now in the /web/ folder.